guide/designing-products/lightning-service-providers.md

```diff
@@ -0,0 +1,306 @@
+
+A [Lightning address](https://lightningaddress.com/) service gives users a reusable, human-readable way to receive lightning payments, such as bosch@bitcoin.design.
+
+Lightning addresses use a non-custodial, trusted third-party server to forward payment requests. This opens up the possibility that the LSP could censor user payments on the application layer by not relaying them between the parties.
+
+</div>
+
+## Payment forwarding
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/designing-products/lightning-service-providers/payment-forwarding.jpg"
+   retina = "/assets/images/guide/designing-products/lightning-service-providers/payment-forwarding@2x.jpg"
+   alt-text = "Lightning address LSP service."
+   width = 400
+   height = 300
+   layout = "float-right-desktop"
+%}
+
+Lightning payments require a route through the network to be calculated for sending, and a user must be online for receiving. Mobile nodes, especially ones running on lower end devices, may have difficulties calculating a route which can result in higher network fees. They are also regularly offline meaning a user will have to be online to receive a payment.
+
+A payment forwarding service solves these issues by acting as an non-custodial intermediary for payments. The route calculation for sending bitcoin can be conducted by the LSPs node on behalf of the user. And for receiving, if the user is offline, the LSP can hold the payment and forward it once the user comes back online.
+
+[Trampoline payments](https://bitcoinops.org/en/topics/trampoline-payments/) remove the need for a single third-party node to calculate a payment route for a user. However, these are currently not widely supported.
+
+[Phoenix](https://phoenix.acinq.co/) is a non-custodial lighting wallet offering this service.
+
+</div>
+
+## Swap-out
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/designing-products/lightning-service-providers/swap-out.jpg"
+   retina = "/assets/images/guide/designing-products/lightning-service-providers/swap-out@2x.jpg"
+   alt-text = "Swapping out of the lightning network."
+   width = 400
+   height = 300
+   layout = "float-right-desktop"
+%}
+
+Making a payment directly from lightning to on-chain is not possible. A user may want to move funds from their lightning channel to on-chain for more secure storage.
+
+A swap-out service allows users to send bitcoin from lightning to on-chain. Users send their lightning bitcoin to the LSP. They then make the on-chain payment on the users behalf. This can have added privacy benefits as the user is receiving one of the LSPs UTXOs and not the ones in their channel.
```

Comment:
```suggestion
A swap-out service allows users to send bitcoin from lightning to on-chain. Users send their lightning bitcoin to the LSP. They then make the on-chain payment on the user's behalf. This can have added privacy benefits as the user is receiving one of the LSPs UTXOs and not the ones in their channel.
```